_develop/connect/overview.md

```diff
@@ -8,22 +8,48 @@ permalink: /:collection/:path.html
 * TOC
```

Comment:
I find it confusing that the URL of the file doesn't line up with the title. Introduction > "overview.html".

Similar: "Guide" > "get-started.html"

Reply:
Agreed, I'll modify these to align.